README.md

```diff
@@ -1 +1,27 @@
-# magento-cloud-patches
\ No newline at end of file
+# Magento Cloud Patches
+
+## Welcome
+Magento Cloud Patches is a set of patches, previously used within ECE-Tools to improve integration of all Magento versions with Cloud and deliver critical fixes on the fly. Starting with ece-tools `2002.0.22`/`2002.1.0`, this package is required to provide same patching flow as in ece-tools `2002.0.21` and earlier versions.
+
+## Contributing to Magento Cloud Patches Code Base
+You can submit pull requests to fix potential bugs. If you find a bug or have a new suggestion, let us know by creating a Github issue.
+
+*Please note:* this repository is not an official support channel. To get project-specific help, please create support ticket through [Support Portal](https://support.magento.com). Support-related issues will be closed with the request to open a support ticket.
+
+# Magento Cloud Suite
+The Magento Cloud Suite includes a set of packages designed to deploy and manage Magento Commerce installations on the Cloud platform.
+- The [ece-tools package](https://github.com/magento/ece-tools) - A set of scripts and tools designed to manage and deploy Cloud projects
+- [Magento Cloud Components](https://github.com/magento/magento-cloud-components) package - Extended Magento Commerce core functionality for sites deployed on the Cloud platform
+- [Magento Cloud Docker](https://github.com/magento/magento-cloud-docker) package - Functionality and Docker images to deploy Magento Commerce to a local Cloud environment
+- [Magento Cloud Patches](https://github.com/magento/magento-cloud-patches) package - Set of patches which improve integration of all Magento versions with Cloud environments
```

Comment:
```suggestion
- [Magento Cloud Patches](https://github.com/magento/magento-cloud-patches) package - Set of patches which improve the integration of all Magento versions with Cloud environments.
```

Reply:
Don't place a period after it if none of the others in the list have a period.